py/pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+"setuptools>=42",
+"wheel"
```

Comment:
I actually had the `build` package missing (when `python3 -m build`), but I'm guessing that package might use this file as an input, so it wouldn't make sense to list `build` here?

Reply:
I don't think you even need `wheel`, just having `build` should be enough.

Reply:
Added a line to install the `build` package in the README before doing `python3 -m build`